wiki/advantages-and-features.md

```diff
@@ -23,14 +23,20 @@ things) see:
 * [Comparison to Primecoin](#comparison-to-primecoin "wikilink")
 * [Comparison to Tokens](#comparison-to-tokens "wikilink")
 
-Additionally, Gridcoin's system of scrapers offers a scalable system that 
-minimizes that amount of centralization. Scrapers get BOINC statistics 
+Additionally, Gridcoin's system of scrapers (often called "oracles") offers a scalable system that 
+minimizes that amount of centralization. In fact, it was one of the first cryptocurrencies
+to implement a federated oracle system. Scrapers get BOINC statistics 
```

Comment:
> In fact, it was one of the first cryptocurrencies
to implement a federated oracle system

this claim seems suspect

Reply:
Were we not? "One of the first" is also an accurate claim even if we were 150th.

Reply:
Gridcoin did not implement scrapers until 2018. We were far from the first

Reply:
Fixed